MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+include README.md
+include vroom/VERSION.txt
```

Comment:
Oops, version on PyPI is broken and apparently needed this line. Not sure why vimdoc didn't, but I updated for both projects.

Reply:
Actually updated to use setuptools-scm instead and removed it again.